documentation/modules/upgrading/proc-upgrade-kafka-kraft.adoc

```diff
@@ -0,0 +1,130 @@
+// This module is included in the following assemblies:
+//
+// assembly-upgrade.adoc
+
+[id='proc-upgrade-kafka-kraft-{context}']
+= Upgrading KRaft-based Kafka clusters and client applications
+
+[role="_abstract"]
+Upgrade a KRaft-based Strimzi Kafka cluster to the latest supported Kafka version and KRaft metadata version.
+
+You should also choose a xref:con-strategies-for-upgrading-clients-{context}[strategy for upgrading clients].
+Kafka clients are upgraded in step 6 of this procedure.
+
+NOTE: Refer to the Apache Kafka documentation for the latest on support for KRaft-based upgrades.
+
+.Prerequisites
+
+* The Cluster Operator is up and running.
+* Before you upgrade the Strimzi Kafka cluster, check that the `Kafka.spec.kafka.config` properties of the `Kafka` resource do _not_ contain configuration options that are not supported in the new Kafka version.
+
+.Procedure
+
+. Update the Kafka cluster configuration:
++
+[source,shell,subs=+quotes]
+----
+kubectl edit kafka <kafka_configuration_file>
+----
+
+. If configured, check that the `spec.kafka.metadataVersion` is set to the _current_ version.
```

Comment:
I do not think it is clear what _current_ means. At this point, it should be normally set to a metadata version supported by both the old version (we use today) and new version (we want to upgrade to).

Reply:
changed to:   If configured, check that the current `spec.kafka.metadataVersion` is set to a version supported by the version of Kafka you are upgrading to.